source/GameWindow.cpp

```diff
@@ -79,6 +79,10 @@ bool GameWindow::Init(bool headless)
 	// This sets it for both X11 and Wayland.
 	setenv("SDL_VIDEO_X11_WMCLASS", "io.github.endless_sky.endless_sky", true);
 #endif
+#if defined(__linux__) && !defined(ES_STEAM)
```

Comment:
How about adding a new CMake option, say `ES_ENABLE_WAYLAND` defaulted to ON that would enable Wayland support, but then when building the release build we disable it?

Reply:
I guess that should work.